apps/easypid/base.app.config.js

```diff
@@ -0,0 +1,182 @@
+const APP_VARIANT = process.env.APP_VARIANT || 'production'
+
+const variants = {
+  development: {
+    bundle: '.dev',
+    name: ' (Dev)',
+  },
+  preview: {
+    bundle: '.preview',
+    name: ' (Preview)',
+  },
+  production: {
+    bundle: '',
+    name: '',
+  },
+}
+
+const variant = variants[APP_VARIANT]
+
+if (!variant) {
+  throw new Error(`Invalid variant provided: ${process.env.APP_VARIANT}`)
+}
+
+// Common invitation schemes
+const baseInvitationSchemes = [
+  'openid',
+  'openid-initiate-issuance',
+  'openid-credential-offer',
+  'openid-vc',
+  'openid4vp',
```

Comment:
I just added a pr a few days ago that adds new schemes. Can we integrate that here as well?